Resources/Prototypes/_LostParadise/Entities/Objects/Consumable/Drinks/drinks_bottles.yml

```
@@ -0,0 +1,247 @@
+- type: entity
+  parent: [DrinkBottleVisualsOpenable, DrinkBottlePlasticBaseFull]
+  id: LPPDrinkCamomileTeaBottleFull
+  name: camomile tea bottle
+  description: Bottle of chamomile tea is the perfect remedy for your peace of mind.
+  components:
+  - type: Item
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)